srcs/parse/initialize_lexer.c

```diff
@@ -20,12 +20,18 @@ int	max_token_num(char *str)
 	return (token_num);
 }
 
-t_token	*initialize_lexer(char *str)
+t_lexer_manager	*initialize_lexer(char *str)
```

Comment:
initializeしてるのは`lexer_manager`かも。とはいえ構造もまだ変わりそうなので今は変えなくて良いと思ってます。
```suggestion
t_lexer_manager	*initialize_lexer_manager(char *str)
```